mysql/server/database.sls

```diff
@@ -1,11 +1,22 @@
 {%- from "mysql/map.jinja" import server with context %}
+{%- if server.admin is defined %}
```

Comment:
Hi @fpytloun,

Yeah sorry you're totally right, I was doing monkey patching :).

Thanks!
I think this new approach is much cleaner, what do you think?

Thanks!